src/encoded/schemas/file.json

```diff
@@ -388,6 +390,39 @@
                 "revoked",
                 "released"
                 ]
+        },
+        "dbxrefs": {
+            "@type": "@id",
+            "rdfs:subPropertyOf": "rdfs:seeAlso",
+            "title": "External identifiers",
+            "description": "Unique identifiers from external resources.",
+            "type": "array",
+            "default": [],
+            "items": {
+                "title": "External identifier",
+                "description": "A unique identifier from external resource.",
+                "type":  "string",
+                "pattern": "^\\S+:\\S+"
+            }
+        },
+        "pipeline": {
+            "title": "Pipeline",
+            "description": "The pipeline the file was generated from.",
+            "comment": "For pipelines, see pipeline.json for available identifiers.",
+            "type": "string",
+            "linkTo": "pipeline"
+        },
+        "steps": {
+            "title": "Analysis steps" ,
+            "description": "The runs of the step the file was generated from.",
+            "type": "array",
+            "items": {
+                "title": "Analysis step",
+                "description": "The run instance of the step the file was generated from.",
+                "comment": "For steps, see analysis_step_run.json (and analysis_step.json) - may be virtual",
+                "type": "string",
+                "linkTo": "analysis_step_run"
+            }
```

Comment:
@lrowe This is to address fantom analysis_step_run that generate files that we don't capture from uniform pipeline but are runs. An example is the methylation filtering step, which we don't capture the filtered fastq's but they are the inputs into the alignments step.

As for the pipeline, ya we can remove that from file schema I think. Was an artifact of an old version of the pipeline.